buildchain/buildchain/iso.py

```diff
@@ -107,7 +107,53 @@
         destination_directory=constants.ISO_ROOT,
         source_prefix=Path('scripts'),
         task_dep=['_iso_mkdir_root']
-    )
+    ),
+    helper.FileTree(
+        basename='_iso_add_tree',
+        files=(
+            Path('metalk8s'),
+
+            Path('metalk8s_cli/__init__.py'),
```

Comment:
Not sure this should be in the project root. Is this *the* CLI to work with MetalK8s? I.e., will it include the stuff we wanted to put in the `kubectl` plugin?

Reply:
I think we can make it _the_ CLI for MetalK8s, and as a Python package, export `entrypoints` for some commands we want to install/use as `kubectl` plugin(s). Would this be a problem?
I also don't see why it should not be an installable/shareable package, as `kubectl` plugins would be.